packages/e2e-test-utils/src/uploadFile.js

```diff
@@ -20,8 +20,11 @@
 import { join, extname, resolve } from 'path';
 import { tmpdir } from 'os';
 import { copyFileSync } from 'fs';
-import { v4 as uuid } from 'uuid';
 
+const hashCode = (s) =>
+  s.split('').reduce((a, b) => ((a << 5) - a + b.charCodeAt(0)) | 0, 0);
```

Comment:
This looks like it was copied from somewhere, seems a bit magic without an explanation of what this does.

Why can't we just use the file name as-is without doing any such magic?

Reply:
I don't know, it doesn't seem to like, which is why I closed the other PR and opened this.

Reply:
Copied from https://stackoverflow.com/questions/7616461/generate-a-hash-from-string-in-javascript

Reply:
We can't just copy code from SO for copyright/license reasons.